app/src/main/java/com/asfoundation/wallet/transactions/Transaction.java

```diff
@@ -244,8 +242,6 @@ static TransactionType fromInt(int type) {
 
     static TransactionStatus fromInt(int status) {
       switch (status) {
-        case 0:
-          return SUCCESS;
         case 1:
           return FAILED;
         case 2:
```

Comment:
There are some methods that can be replaced by Object.equals: approveTransactionId != null ? !approveTransactionId.equals(that.approveTransactionId)
        : that.approveTransactionId != null --> !Objects.equals(approveTransactionId, that.approveTransactionId)